environment-dev.yml

```diff
@@ -11,7 +11,7 @@ dependencies:
 - importlib-metadata != 8.7.0 # conda-forge version is broken
 - jsonpickle >=3.1.0
 - numba >=0.57.0
-- numpy >=1.24.0
+- numpy >=1.25.0
```

Comment:
Is this related to some error?

Reply:
Nope, just weaning ourselves off of NumPy v1 gradually. See: https://scientific-python.org/specs/spec-0000/